css/home-page.css

```diff
@@ -0,0 +1,173 @@
+.home-page{
+    background-color: lightgoldenrodyellow;
+}
+
+.home-header{
+    align-items: center;
+    background-color: gray;
+    border: 2px solid black;
+    border-radius: 5px;
+    border-radius: 5px;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap; 
+    gap: 20%;
+    min-height: 160px;
+    justify-content:space-evenly;
+}
+
+.header-img{
+    border: 2px solid black;
+    border-radius: 5px;
+    min-width: 300px;
+    margin: 10px 0;
+}
+
+.header-content{
+    display: flex;
+    flex-direction: row;
+    gap: 50px;
+    padding: 50px 20;
+    justify-content: flex-start;
+
+}
+
+.navbar-container{
+    background-color: gray;
+    border: 2px solid black;
+    border-radius: 5px;
+    float: left;
+    margin-left: 40px;
+    margin-top: 20px;   
+    width: 200px;
+
+    
+}
+
+.home-navbar {
+    list-style-type: none;
+    padding: 0;
+}
+
+.nav-branch {
+    position: relative;
+}
+
+.nav-links {
+    color: #000;
+    display: block;
+    padding: 10px 15px;
+    text-decoration: none;
+}
+
+.items-text {
+    display: none;
+}
+
+.sub-navbar {
+    background-color: lightgray;
+    border: 2px solid black;
+    border-radius: 5px;
+    box-shadow: 0 8px 16px rgba(0, 0, 0, 0.2);
+    display: none;
+    left: 184px;
+    list-style-type: none;
+    padding: 0;
+    position: absolute;
+    top: 10%;
+}
+
+.sub-nav-links {
+    display: block;
+    padding: 10px 15px;
+    text-decoration: none;
+    color: #000;
+}
+
+.nav-branch:hover .sub-navbar {
+    display: block;
+}
+
+.home-page-content{
+    margin-left: 300px;
+    margin-top: 50px;
+}
+
+.home-product{
+    background-color: white;
+    border: 1px solid black;
+    float: left;
+    margin: 100px 100px;
+    height: 475px;
+    width: 300px;
+}
+
+.img-product{
+    height: 250px;
+    margin: 20px 25px;
+    width: 250px;
+}
+
+.product-title{
+    margin: 0 10%;
+}
+
+.description-container{
+    border: 1px solid black;
+    margin: 18px 10%;
+    height: 80px;
+    width: 250px;
```

Comment:
Here you are applying the same image width to the description.
Try to find a different way:
- set the padding to the main container;
- display inner elements in column using flex;
- search how to create a squared container that can be resized keeping the right aspect ratio;
- Set a max width to the container and let it expand (flex-grow);

Reply:
fixed